src/components/BCHTransaction.vue

```diff
@@ -51,7 +96,8 @@
         </tr>
       </tbody>
     </table>
-    <h2>Inputs</h2>
+    <h2 v-if="transactionData['inputs'].length === 1">1 Txn Input</h2>
```

Comment:
NIT: Prefer to just use "Inputs" and "Outputs"

Reply:
done.